bower.json

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
Are we going to abandon bower completely? What about other elements that still uses it? https://github.com/Juicy/juicy-jsoneditor/blob/master/bower.json